_schema.graphql

```diff
@@ -1958,7 +1958,7 @@ type Collection implements Node {
     before: String
     last: Int
     private: Boolean = false
-    sort: CollectionSorts = POSITION_DESC
+    sort: CollectionSorts
```

Comment:
These were conscious changes ^ 

Stitching didn't reliably show the right output here, so I didn't want to be relying on the implementation being hooked up correct. Instead. I moved the default setting to be at runtime.

Reply:
Did these defaults already have associated tests?